src/main/java/codechicken/nei/ItemHistoryPanel.java

```diff
@@ -39,8 +39,7 @@ public void update() {
 
     public void addItem(ItemStack stack) {
         if (stack != null) {
-            ItemStack is = stack.copy();
-            is.stackSize = 1;
+            ItemStack is = StackInfo.loadFromNBT(StackInfo.itemStackToNBT(stack), 0);
```

Comment:
Is creating intermediate NBT really necessary?

Reply:
nei does not know and should not know how different mods store the sizes of stacks  (mostly fluid amount). StackInfo provides an interface for mods to standardize stack sizes where this value can be easily changed. There is no other option to qualitatively change the stack size without reference to mods